app/assets/stylesheets/charges.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the charges controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```

Comment:
file deleted from the repository https://github.com/fabriziobertoglio1987/railsmentor/commit/36c8a41b58caa421d02225ca4ef8d03344c32cdb